community/libmustache/APKBUILD

```diff
@@ -0,0 +1,39 @@
+# Contributor: Renoir Boulanger <hello@renoirboulanger.com>
+# Maintainer: John Boehr <jbboehr@gmail.com>
+pkgname=libmustache
+pkgver=0.4.2
+pkgrel=1
+pkgdesc="C++ implementation of Mustache intended mainly for use as a PHP extension"
+url="https://github.com/jbboehr/libmustache"
+arch="all"
+license="MIT"
+depends="libgcc libstdc++"
```

Comment:
These dependencies are automatically detected by abuild (see the Travis build log) and do not have to be specified specifically.